fabric-tool-attribute-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/MiningConstants.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.tool.attribute.v1;
+
+/**
+ * Constants for all the mining levels and speed multipliers of vanilla materials.
+ */
+public final class MiningConstants {
+	private MiningConstants() { }
+
+	public static final int WOOD_MINING_LEVEL = 0;
+	public static final int STONE_MINING_LEVEL = 1;
+	public static final int IRON_MINING_LEVEL = 2;
+	public static final int DIAMOND_MINING_LEVEL = 3;
+	public static final int NETHERITE_MINING_LEVEL = 3;
```

Comment:
Isn't this for 1.15, are we sure that we want netherite here

Reply:
I mean it's just a named constant, so I don't think there's any harm in having it there as reference material.